v2/authorisation/middleware.go

```diff
@@ -142,11 +147,14 @@ func (m PermissionCheckMiddleware) Require(permission string, handlerFunc http.H
 			entityData.UserID = zebedeeIdentityResponse.Identifier
 		}
 
-		attributes := map[string]string{}
-
-		collectionIdAttribute, _ := headers.GetCollectionID(req)
-		if collectionIdAttribute != "" {
-			attributes[collectionIdAttributeKey] = collectionIdAttribute
+		var attributes map[string]string
+		if getAttributes != nil {
```

Comment:
If `getAttributes` is nil should that be an error or do we proceed and let it fail quietly?

Reply:
a nil value is permitted, it is used by the refactored `Require` method